qiskit/pulse/instruction_schedule_map.py

```diff
@@ -404,3 +400,9 @@ def _get_instruction_string(inst: Union[str, Instruction]):
             raise PulseError(
                 'Input "inst" has no attribute "name". This should be a circuit "Instruction".'
             ) from ex
+
+
+def __getattr__(name):
+    from qiskit.pulse import calibration_entries
+
+    return getattr(calibration_entries, name)
```

Comment:
I'm a bit confused by this, why do we need to redirect the getattrs on this `qiskit.pulse.instruction_schedule_map` to the calibration entries module? I don't see anything really using this in the PR.

Reply:
`CalibrationPublisher` is no longer used and I needed to import it for backward compatibility. See [292fabd](https://github.com/Qiskit/qiskit-terra/pull/9597/commits/292fabd7e0b4a42880e8d3283181df1988eb9274)

Reply:
Ah ok that makes sense. I agree https://github.com/Qiskit/qiskit-terra/pull/9597/commits/292fabd7e0b4a42880e8d3283181df1988eb9274 makes that intent clearer too (and also limits us to what we really need for backwards compatibility)